base/linalg/generic.jl

```diff
@@ -800,6 +800,8 @@ condition number, or componentwise relative condition number.
 """
 condskeel(A::AbstractMatrix, x::AbstractVector, p::Real=Inf) = norm(abs.(inv(A))*(abs.(A)*abs.(x)), p)
 
+issymmetric(A::AbstractMatrix{<:Real}) = ishermitian(A)
```

Comment:
If the element type is real, I think defining issymmetric would be more common for user defined array types